test/Babel-typescript.spec.js

```diff
@@ -0,0 +1,62 @@
+import { resolve } from 'path'
+import jestVue from '../vue-jest'
+import {
+  readFileSync,
+  writeFileSync
+} from 'fs'
+
+import clearModule from 'clear-module'
+import cache from '../lib/cache'
+
+beforeEach(() => {
+  cache.flushAll()
+  clearModule.all()
+})
+test('babel 7 configuration expects filename [default: "unknown"] to compile', () => {
```

Comment:
In this case, since it's a small change and difficult to test, you can remove these tests and we'll leave it untested

Reply:
done